File: app/src/main/java/com/lumiyaviewer/lumiya/slproto/events/SLTaskInventoryReceivedEvent.java
package com.lumiyaviewer.lumiya.slproto.events;

import com.lumiyaviewer.lumiya.slproto.inventory.SLTaskInventory;
import java.util.UUID;

public class SLTaskInventoryReceivedEvent {
    public final UUID taskID;
    public final SLTaskInventory taskInventory;

    public SLTaskInventoryReceivedEvent(UUID uuid, SLTaskInventory taskInventory) {
        this.taskID = uuid;
        this.taskInventory = taskInventory;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/slproto/events/SLTaskInventoryReceivedEvent.kt
package com.lumiyaviewer.lumiya.slproto.events

import com.lumiyaviewer.lumiya.slproto.inventory.SLTaskInventory
import java.util.UUID

data class SLTaskInventoryReceivedEvent(
    @JvmField val taskID: UUID,
    @JvmField val taskInventory: SLTaskInventory
)
